### ISSUE TITLE
Question about tf_actions_and_internals in DistributionModel

### ISSUE THREAD

**Issue by mysl** [reinforceio/tensorforce#359]
Hi there,
when I read the code **_tf_actions_and_internals_** in distribution_model.py,  I found that in the implementation, it is assumed distribution.parameterize() returns  list as [logits, probabilities, state_value].  But from the implementation code of distributions, this is true only for Categorical distribution.  And the DistributionModel seems allowing more distributions than that.  Did I miss anything or is it a bug?  

Thanks!

=========================================================    
           # parameterize() returns list as [logits, probabilities, state_value]
            self.network.set_named_tensor(name_prefix + "logits", distr_params[0])
            self.network.set_named_tensor(name_prefix + "probabilities", distr_params[1])
            self.network.set_named_tensor(name_prefix + "state_value", distr_params[2])

**Comment by michaelschaarschmidt** [comment]
Hi,
the other distributions also return things but it's true that these maybe could be named differently here to be more precise

**Comment by mysl** [comment]
@michaelschaarschmidt ,  thanks for reply! 
Then the logic here is to set the first three items of the distr_params regardless what they are?

**Comment by michaelschaarschmidt** [comment]
yes, sorry did not see this
